server/controller/accountsMangment/captains/getCaptains.js

```diff
@@ -0,0 +1,16 @@
+const {
+  getCaptains,
+} = require('../../../database/queries/captain/getCaptains');
+
+const getCaptain = (request, response) => {
+  getCaptains()
+    .then((res) => {
+      if (res.rows) {
+        response.status(200).send({ result: res.rows });
+      }
```

Comment:
@Jamalat-shamallakh It might be a good idea to add an `} else {` and return an error for that case.